Snowflake.Data.Tests/IntegrationTests/ConnectionSinglePoolCacheIT.cs

```diff
@@ -71,7 +92,54 @@ public void TestConnectionPoolFull()
             Assert.AreEqual(ConnectionState.Closed, conn4.State);
             SnowflakeDbConnectionPool.ClearAllPools();
         }
+
+        [Test]
+        public void TestConnectionPoolDisableFromHighLevel()
```

Comment:
more precisely it would be from PoolManagerLevel but I don't push to change, WDYT?

Reply:
done